### PR TITLE
#1114 add new page for Oxyanime.to

### PR DESCRIPTION
**Pull request by karsiong**
#1114 
Oxyanime.to

Feature support
- Overview Page
- Next Episode
- Update Check
- Quick Search

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,6 +300,10 @@ Makes it possible to use your MyAnimeList/Anilist/Kitsu/Simkl anime/mangalist as
                 <td><a href="https://www.animetoast.cc"><img src="https://favicon.malsync.moe/?domain=https://www.animetoast.cc"> Animetoast</a></td>
                 <td></td>
                 <td></td>
+              </tr><tr>
```
Comment (henrik9999) [inline README.md:303]: Readme file is auto generated dont change this

File: pages.md [hunk 2]
```diff
@@ -523,6 +523,13 @@
     <td>:heavy_check_mark:</td>
     <td>:heavy_check_mark:</td>
     <td>:heavy_check_mark:</td>
+  </tr><tr>
```
Comment (henrik9999) [inline pages.md:526]: Pages file is auto generated dont change this

File: src/pages/list.json [hunk 3]
```diff
@@ -688,5 +688,10 @@
     "domain": "https://www.animetoast.cc",
     "type": "anime",
     "name": "Animetoast"
+  },
```
Comment (henrik9999) [inline src/pages/list.json:691]: List file is auto generated dont change this

**Review by henrik9999 — CHANGES_REQUESTED** [review]

File: pages.md (hunk 2 above)
Comment (karsiong) [reply pages.md]: done revise to unchanged

File: README.md (hunk 1 above)
Comment (karsiong) [reply README.md]: done revise to unchanged

**Comment by karsiong** [comment]
ok the md file and list is already remove

**Review by henrik9999 — CHANGES_REQUESTED** [review]
tests are missing

**Comment by karsiong** [comment]
ok just include the test case

**Comment by karsiong** [comment]
I will take all responsible for this page change.

File: src/pages/pages.ts [hunk 4]
```diff
@@ -137,6 +137,7 @@ import { MangaReader } from './MangaReader/main';
 import { AnimeOnsen } from './AnimeOnsen/main';
 import { Puray } from './Puray/main';
 import { Animetoast } from './Animetoast/main';
+import { OxyAnime } from './OxyAnime/main';
```
Comment (lolamtisch) [inline src/pages/pages.ts:140]: The import path is wrong

**Review by lolamtisch — CHANGES_REQUESTED** [review]
Please run `npm run lint:fix` and rebase to master

File: src/pages/pages.ts (hunk 4 above)
Comment (karsiong) [reply src/pages/pages.ts:140]: ok

**Comment by karsiong** [comment]
the conflicts part not sure how to solve it, it due to before after commit.

**Comment by karsiong** [comment]
ah.. that 1 mistake
should
` return title.replace(/\-/g,' ');`

sorry

**Comment by karsiong** [comment]
Requested approval for test workflow

**Comment by karsiong** [comment]
i have tested locally may time is just work....hmm

**Comment by karsiong** [comment]
the test use puppeteer? it able to load js framework web?

**Comment by karsiong** [comment]
hmm... missing episode parameter in tests.json
Sorry

**Comment by lolamtisch** [comment]
Unfortunately after looking deeper into the website, there are too many problematic things and we decided against adding support for this website to malsync.

Some of the points:
- missuse of github for hosting
- Requesting gogoanime directly from the client
- Exclusively using gogoanime, myanimelist and imgur for image hosting
- Mostly only using other known gogoanime scrapper apis
- Bad practice (random w3school css files)
- Dev-tools block script

In general the problem here is the **extensive** misuse of others peoples infrastructure, which is a core problem that can not easily be solved.

Sorry that you already put so much efford into this implementation. I should have checked the page before.

**Pull request closed without merging**